lldb/unittests/Host/HostTest.cpp

```diff
@@ -90,6 +92,30 @@ TEST(Host, LaunchProcessSetsArgv0) {
   ASSERT_THAT(exit_status.get_future().get(), 0);
 }
 
+TEST(Host, FindProcesses) {
+  SubsystemRAII<FileSystem, HostInfo> subsystems;
+
+  if (test_arg != 0)
+    exit(0);
```

Comment:
```suggestion
  if (test_arg != 0) {
    // Give the parent time to retrieve information about us. It will kill us when it is done.
    std::this_thread::sleep_for(std::chrono::seconds(10));
    exit(0);
  }
```